Source/FieldSolver/WarpXPushFieldsEM.cpp

```diff
@@ -492,6 +492,8 @@ WarpX::ApplyInverseVolumeScalingToCurrentDensity (MultiFab* Jx, MultiFab* Jy, Mu
     const std::array<Real,3>& dx = WarpX::CellSize(lev);
     const Real dr = dx[0];
 
+    constexpr int NODE = amrex::IndexType::NODE;
```

Comment:
Could we add a test like the following?
```suggestion
    constexpr int NODE = amrex::IndexType::NODE;
    AMREX_ALWAYS_ASSERT_WITH_MESSAGE(Jx->ixType().ixType()[0] != NODE,
        "Jr should never node-centered in r");
```
Also, why are these called `Jx` etc.? Note that this shouldn't be changed in this PR anyway.

Reply:
Adding the assert is a good idea, though is unlikely to ever be violated. I'll commit the change.

I kept the names Jx to Jy to keep the connection with the names of the arrays that are passed in. Changing them though would be fine.

Reply:
Sounds good, thanks.